ssc/cmod_geothermal_costs.cpp

```diff
@@ -778,10 +778,11 @@ class cm_geothermal_costs : public compute_module
         //OM Cost calculations
         /*
         double unit_plant = as_double("gross_output");
-        double cooling_water_flow_rate = as_double("cooling_water_flow_rate"); //todo provide input
-        double cooling_water_cost = as_double("cooling_water_cost"); //todo provide input
+        double cooling_water_flow_rate = as_double("cwflow"); //todo provide input
+        double cooling_water_cost = 300; //$/acre-ft //todo provide input
```

Comment:
Are we going to get an input value from GTO or is the cooling water cost the same for all scenarios?

Reply:
Updated to be 1.7 * chemical_ppi for flash, overall costs goes to 0 for binary.